x-pack/plugins/cross_cluster_replication/public/app/sections/home/home.js

```diff
@@ -20,90 +20,154 @@ import {
   EuiPageBody,
   EuiPageContent,
   EuiSpacer,
+  EuiTab,
+  EuiTabs,
   EuiText,
   EuiTitle,
 } from '@elastic/eui';
 
 import { listBreadcrumb } from '../../services/breadcrumbs';
 import routing from '../../services/routing';
 import { AutoFollowPatternList } from './auto_follow_pattern_list';
+import { FollowerIndicesList } from './follower_indices_list';
 import { SectionUnauthorized } from '../../components';
 
 export const CrossClusterReplicationHome = injectI18n(
   class extends PureComponent {
     static propTypes = {
       autoFollowPatterns: PropTypes.array,
+      isAutoFollowApiAuthorized: PropTypes.bool,
+      followerIndices: PropTypes.array,
+      isFollowerIndexApiAuthorized: PropTypes.bool,
     }
 
     state = {
-      sectionActive: 'auto-follow'
+      activeSection: 'follower_indices'
     }
 
+    tabs = [{
+      id: 'follower_indices',
+      name: (
+        <FormattedMessage
+          id="xpack.crossClusterReplication.autoFollowPatternList.followerIndicesTitle"
+          defaultMessage="Follower indices"
+        />
+      )
+    }, {
+      id: 'auto_follow_patterns',
+      name: (
+        <FormattedMessage
+          id="xpack.crossClusterReplication.autoFollowPatternList.autoFollowPatternsTitle"
+          defaultMessage="Auto-follow patterns"
+        />
+      )
+    }]
+
     componentDidMount() {
+      const { match: { params: { section } } } = this.props;
```

Comment:
You don't need to add those lines as `getDerivedStateFromProps` will handle it

Reply:
good catch, leftover from a previous attempt